themes/digital.gov/layouts/partials/core/usa-collection.html

```diff
@@ -26,6 +26,7 @@
         {{ if $source.Params.logo }}
           {{ $logo_source = (printf "/logos/%s%s" $source.Params.logo "-icon.png") }}
         {{ end }}
+        {{ $href = .Params.source_url }}
```

Comment:
Setting the `$href` in the first condition check ensures the hugo resource has the correct link url.